server/models/db/student.js

```diff
@@ -10,7 +10,21 @@ const Tables = require('../../constants/tables.json');
 async function getStudent(email, password) {
     return userModel.getUser(email, password, Tables.student, Student);
 }
+// This method inserts a student account given specific information.
+async function insertStudentAccount(firstName, lastName, type, studentID, email, password, phone, school_id ) {
+    //Data recieved from the front end sign up form
+ //Insert one row into a table
```

Comment:
Hi Nancy,
To increase code readability, I would recommend indenting this comment or using right click > Format in Visual Studio Code to ensure that the file formatting is per standard.

Reply:
Hi Amy,
Thank you for suggesting that!

My action: I've formatted the code to align with the accepted coding standards in server/models/db>student.js